### PR TITLE
Update CHANGELOG.md for the 2.3.3 release

### PR DESCRIPTION
**Pull request by daschuer**
I think it is time for a release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.3) (unreleased)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:3]: 2.3.3

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+* Pioneer-DDJ-SB3: fix shift button release [#4659](https://github.com/mixxxdj/mixxx/pull/4659)
+* Traktor S3: Push two deck switches to explicitly clone decks [#4665](https://github.com/mixxxdj/mixxx/pull/4665) [#4671](https://github.com/mixxxdj/mixxx/pull/4671) [lp:1960680](https://bugs.launchpad.net/mixxx/+bug/1960680)
+* Behringer DDM4000: Update controller mapping [#4318](https://github.com/mixxxdj/mixxx/pull/4318)
+* MC7000: Fix 'inverted shift' bug in the controller mapping [#4755](https://github.com/mixxxdj/mixxx/pull/4755)
+* Fix spinback and break effect in the controller engine [#4708](https://github.com/mixxxdj/mixxx/pull/4708)
+* Fix scratch on first wheel touch [#4761](https://github.com/mixxxdj/mixxx/pull/4761) [lp:1800343](https://bugs.launchpad.net/mixxx/+bug/1800343)
+* Pref controller: prevent false-positive 'dirty' flag [#4721](https://github.com/mixxxdj/mixxx/pull/4721) [lp:1920844](https://bugs.launchpad.net/mixxx/+bug/1920844)
+* Fix rare SEGFAULT when closing the progress dialog [#4695](https://github.com/mixxxdj/mixxx/pull/4695)
```
Comment (Holzhaus) [inline CHANGELOG.md:12]:
```suggestion
* Fix rare crash when closing the progress dialog [#4695](https://github.com/mixxxdj/mixxx/pull/4695)
```

No regular user knows what a segfault is, at least not on windows or macOS.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.3) (unreleased)
+
+* Pioneer-DDJ-SB3: fix shift button release [#4659](https://github.com/mixxxdj/mixxx/pull/4659)
```
Comment (Holzhaus) [inline CHANGELOG.md:5]: What does this mean? How was it broken? What works now what didn't work before?

Also, please capitalize the first character after the colon.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+* Traktor S3: Push two deck switches to explicitly clone decks [#4665](https://github.com/mixxxdj/mixxx/pull/4665) [#4671](https://github.com/mixxxdj/mixxx/pull/4671) [lp:1960680](https://bugs.launchpad.net/mixxx/+bug/1960680)
+* Behringer DDM4000: Update controller mapping [#4318](https://github.com/mixxxdj/mixxx/pull/4318)
```
Comment (Holzhaus) [inline CHANGELOG.md:7]: What changed?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,27 @@
...
+* MC7000: Fix 'inverted shift' bug in the controller mapping [#4755](https://github.com/mixxxdj/mixxx/pull/4755)
+* Fix spinback and break effect in the controller engine [#4708](https://github.com/mixxxdj/mixxx/pull/4708)
+* Fix scratch on first wheel touch [#4761](https://github.com/mixxxdj/mixxx/pull/4761) [lp:1800343](https://bugs.launchpad.net/mixxx/+bug/1800343)
+* Pref controller: prevent false-positive 'dirty' flag [#4721](https://github.com/mixxxdj/mixxx/pull/4721) [lp:1920844](https://bugs.launchpad.net/mixxx/+bug/1920844)
+* Fix rare SEGFAULT when closing the progress dialog [#4695](https://github.com/mixxxdj/mixxx/pull/4695)
+* Prevent preferences dialog from going out of screen [#4613](https://github.com/mixxxdj/mixxx/pull/4613)
+* Fix undesired jump-cuts in Auto-DJ [#4693](https://github.com/mixxxdj/mixxx/pull/4693) [lp:1948975](https://bugs.launchpad.net/mixxx/+bug/1948975) [lp:1893197](https://bugs.launchpad.net/mixxx/+bug/1893197)
+* Fix stopped Auto-DJ [#4698](https://github.com/mixxxdj/mixxx/pull/4698) [lp:1893197](https://bugs.launchpad.net/mixxx/+bug/1893197) [lp:1961970](https://bugs.launchpad.net/mixxx/+bug/1961970)
```
Comment (Holzhaus) [inline CHANGELOG.md:15]: This sounds wrong. How about "Fix bug that caused Auto DJ to stop playback after some time"

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,27 @@
...
+* Do not reset xfader when Auto-DJ is deactivated [#4714](https://github.com/mixxxdj/mixxx/pull/4714) [lp:1965298](https://bugs.launchpad.net/bugs/1965298)
+* Change the minimum Auto-DJ transition time to -99 [#4768](https://github.com/mixxxdj/mixxx/pull/4768) [lp:1975552](https://bugs.launchpad.net/mixxx/+bug/1975552)
```
Comment (Holzhaus) [inline CHANGELOG.md:17]: You used "Auto-DJ" but in the manual it's called "Auto DJ" without the dash. We should be consistent IMHO.

**Review by Holzhaus — COMMENTED** [review]
Thanks, some comments.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,27 @@
...
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.3) (unreleased)
+
+* Pioneer-DDJ-SB3: fix shift button release [#4659](https://github.com/mixxxdj/mixxx/pull/4659)
+* Traktor S3: Push two deck switches to explicitly clone decks [#4665](https://github.com/mixxxdj/mixxx/pull/4665) [#4671](https://github.com/mixxxdj/mixxx/pull/4671) [lp:1960680](https://bugs.launchpad.net/mixxx/+bug/1960680)
+* Behringer DDM4000: Update controller mapping [#4318](https://github.com/mixxxdj/mixxx/pull/4318)
+* MC7000: Fix 'inverted shift' bug in the controller mapping [#4755](https://github.com/mixxxdj/mixxx/pull/4755)
+* Fix spinback and break effect in the controller engine [#4708](https://github.com/mixxxdj/mixxx/pull/4708)
+* Fix scratch on first wheel touch [#4761](https://github.com/mixxxdj/mixxx/pull/4761) [lp:1800343](https://bugs.launchpad.net/mixxx/+bug/1800343)
+* Pref controller: prevent false-positive 'dirty' flag [#4721](https://github.com/mixxxdj/mixxx/pull/4721) [lp:1920844](https://bugs.launchpad.net/mixxx/+bug/1920844)
```
[the remark this answers is not in the document]
Comment (ronso0) [reply CHANGELOG.md:11]: "Controller Preferences: Prevent false-positive detection of pending changes"
It's shorter.
Or do you think "false-positive" is already too technical for regular users?

File: CHANGELOG.md (hunk 4 above)
Comment (Swiftb0y) [reply CHANGELOG.md:7]: I don't think its super important to list that here. People can look at the PR description if they're interested

File: CHANGELOG.md (hunk 1 above)
Comment (Swiftb0y) [reply CHANGELOG.md:3]: Do we remove the `(unreleased)` here or does that the changelog updater tool for us? I forgor :skull:

File: CHANGELOG.md (hunk 2 above)
Comment (uklotzde) [reply CHANGELOG.md:12]: "Stability improvements and fixes." Just copy it from any other changelog. :rofl:

File: CHANGELOG.md (hunk 1 above)
Comment (daschuer) [reply CHANGELOG.md:3]: This is a manual step during the release process. I forgot it last time. This will not happen again.

**Comment by ronso0** [comment]
please add recently merged #4789 + #4791
`* Fix passthrough-related crash in waveform code [#4789](https://github.com/mixxxdj/mixxx/pull/4789) [#4791](https://github.com/mixxxdj/mixxx/pull/4791) [lp:1959489](https://bugs.launchpad.net/mixxx/+bug/1959489) [lp:1977662](https://bugs.launchpad.net/mixxx/+bug/1977662)`

**Comment by daschuer** [comment]
I think all comments are addressed.

File: CHANGELOG.md (hunk 1 above)
Comment (Swiftb0y) [reply CHANGELOG.md]: So should the `(unreleased)` be removed here or at some point later? I'm still not familiar how our release process works to be honest.

Comment (daschuer) [reply CHANGELOG.md]: This will be dine during the manual gagging process.

Comment (Swiftb0y) [reply CHANGELOG.md]: Ok thank you.

**Review by Swiftb0y — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Holzhaus) [reply CHANGELOG.md]: I wish there was a `pre-tag` git hook that could check those things and warn if we forgot something.

Comment (Swiftb0y) [reply CHANGELOG.md]: Since you "upload" tags to the remote using `git push <remote> <tagname>`, maybe you could filter for tags in the `pre-push` hook?

**Pull request merged**